jsx/Form.js

```diff
@@ -132,20 +109,52 @@ var FormElement = React.createClass({
       </form>
     );
   }
-});
+}
+
+FormElement.propTypes = {
+  name: PropTypes.string.isRequired,
+  id: PropTypes.string,
+  method: PropTypes.oneOf(['POST', 'GET']),
+  action: PropTypes.string,
+  class: PropTypes.string,
+  columns: PropTypes.number,
+  formElements: PropTypes.shape({
+    elementName: PropTypes.shape({
+      name: PropTypes.string,
+      type: PropTypes.string
+    })
+  }),
+  onSubmit: PropTypes.func,
+  onUserInput: PropTypes.func
+};
+
+FormElement.defaultProps = {
```

Comment:
I'm assuming that by setting defaultProps in each element class you no longer have to overwrite the getDefaultProps function? pls confirm